image-loader/conf/routes

```diff
@@ -3,6 +3,12 @@ POST    /images                     controllers.ImageLoaderController.loadImage(
 POST    /imports                    controllers.ImageLoaderController.importImage(uri: String, uploadedBy: Option[String], identifiers: Option[String], uploadTime: Option[String], filename: Option[String])
 GET     /images/project/:imageId    controllers.ImageLoaderController.projectImageBy(imageId: String)
 
+# Upload Status
+GET     /uploadStatus/:imageId      controllers.UploadStatusController.getUploadStatus(imageId: String)
+POST    /uploadStatus               controllers.UploadStatusController.updateUploadStatus
```

Comment:
I would suggest that the URI for this POST mirrors GET (i.e. `/uploadStatus/:imageId`). If this endpoint then only takes the status and optional error in the JSON body then you'll have the imageId in the path.